include/camp/defines.hpp

```diff
@@ -94,9 +94,11 @@ namespace camp
 #define CAMP_SUPPRESS_HD_WARN
 #endif
 
+#if defined(ENABLE_TARGET_OPENMP)
 #if _OPENMP >= 201511
 #define CAMP_HAVE_OMP_OFFLOAD 1
 #endif
+#endif
```

Comment:
Should enabling target offload with a compiler that does not support it produce an error or fallback?

Reply:
It will inherently produce compilation errors due to the lack of support for the advanced OpenMP offload clauses. I can add a printed warning here though.

Personally, I would prefer to see the compilation errors rather than falling back. For instance, if we fallback to standard OpenMP, the user may think they're just running a very slow OpenMP offload code, and may embark on a misguided journey in to performance debugging.